### ISSUE TITLE
Enable keyboard shortcut to navigate between skill pages while taking a training

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#3329]
This will allow easier navigation between skills in a subject without using the next/previous buttons explicitly. This is particularly impactful for kills with lengthily descriptions, videos and/or prerequisites. Keyboard shortcut should be configurable on the user's preference page.

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/3332

**Comment by dwalizer** [comment]
👍
